larpandoracontent/LArThreeDReco/LArTwoViewMatching/TwoViewLongTracksTool.h

```diff
@@ -0,0 +1,80 @@
+/**
+ *  @file   larpandoracontent/LArThreeDReco/LArTwoViewMatching/TwoViewLongTracksTool.h
+ *
+ *  @brief  Header file for the long tracks tool class.
+ *
+ *  $Log: $
+ */
+#ifndef TWO_VIEW_LONG_TRACKS_TOOL_H
+#define TWO_VIEW_LONG_TRACKS_TOOL_H 1
+
+#include "larpandoracontent/LArThreeDReco/LArTwoViewMatching/TwoViewTransverseTracksAlgorithm.h"
+
+namespace lar_content
+{
+
+/**
+ *  @brief  TwoViewLongTracksTool class
+ */
+class TwoViewLongTracksTool : public TransverseMatrixTool
+{
+public:
+    /**
+     *  @brief  Default constructor
+     */
+    TwoViewLongTracksTool();
+
+    /**
+     *  @brief  Whether a long element shares clusters with any other long elements
+     *
+     *  @param  iIter specifies the long element under consideration
+     *  @param  iteratorList list of iterators to other long elements
+     *
+     *  @return boolean
+     */
+    static bool HasLongDirectConnections(IteratorList::const_iterator iIter, const IteratorList &iteratorList);
+
+    /**
+     *  @brief  Whether a long element is significantly longer that other elements with which it shares a cluster
+     *
+     *  @param  iIter specifies the long element under consideration
+     *  @param  elementList the full list of connected matrix elements
+     *  @param  minMatchedSamplingPointRatio the min ratio between 1st and 2nd highest msps for simple ambiguity resolution
+     *  @param  usedClusters the list of clusters already marked as to be added to a pfo
+     */
+    static bool IsLongerThanDirectConnections(IteratorList::const_iterator iIter, const MatrixType::ElementList &elementList,
+        const unsigned int minMatchedSamplingPointRatio, const pandora::ClusterSet &usedClusters);
```

Comment:
See above comments about dropping `minMatchedSamplingPointRatio` as argument

Reply:
See my answers above! We decided to leave this argument in.